app/pages/docs/tradeoffs.mdx

```diff
@@ -1,58 +1,56 @@
+- Kan je direct toegang krijgen tot de automatisch gegenereerde API-endpoints en kan je ze gebruiken in elke mobiele app (we hebben momenteel een paar mensen die dit doen)
+- Kan je alle gegenereerde endpoints in je app zien met het [`blitz routes`](./cli-routes) commando
+- Kan je de RPC-specificatie voor de endpoints hier zien: [RPC Specification](./rpc-specification)
 
-**What we plan to do going forward:**
+**Wat we nog van plan zijn om te doen:**
 
-1. First step, add docs and tools for making it easier to manually use the generated API endpoints. See https://github.com/blitz-js/blitz/discussions/1907 and https://github.com/blitz-js/blitz/issues/745
-2. Next, add ability to generate a JS/TS client library as a nice way to use the generated endpoints from an app
-3. Next, add some sort of first-class react-native support where you can import blitz queries/mutations directly into your react-native code and have it magically work exactly like it does today with your web app. After the 1.0 release, this will be Brandon's main focus for the year.
+1. Eerst documentatie en tools toevoegen die het eenvoudiger maken om de handmatig gegenereerde API-endpoints te gebruiken. Zie: https://github.com/blitz-js/blitz/discussions/1907 en https://github.com/blitz-js/blitz/issues/745
+2. Vervolgens de mogelijkheid toevoegen om een JS/TS client library te genereren zodat men de gegenereerde endpoints op een gebruiksvriendelijke manier kan gebruiken in een app.
+3. Ten slotte een first-class react-native ondersteuning bieden waardoor je Blitz queries/mutations rechtstreeks in je react-native code kan importeren zodat het net op dezelfde magische wijze werkt als in een huidige web app.
 
 
-## Advanced Backend Architecture {#advanced-backend-architecture}
+## Geavanceerde backendarchitectuur {#advanced-backend-architecture}
 
-Currently Blitz is fairly minimal on backend architecture, especially compared so something like Nest.js or AdonisJs. However, that doesn't mean you can't use those patterns in Blitz, it means you have to set it up yourself. In fact you can use Nest.js in your Blitz app if you need.
+Blitz heeft momenteel een redelijk beperkte backendarchitectuur in vergelijking met andere frameworks zoals NestJS of AdonisJS. Dit betekent echter niet dat je deze patronen niet kan gebruiken in Blitz, 
+maar dan moet je ze wel zelf opzetten. Je kan zelfs NestJS als backend voor je Blitz applicatie gebruiken als je dat wil.
 
-But we are very interested in bringing more advanced backend architectures to Blitz by default or via simple opt-in. We have an [ongoing discussion](https://github.com/blitz-js/blitz/discussions/1841) that we'd love you to chime in on if you have ideas.
+Maar we zijn zeer geïnteresseerd om geavanceerdere backendarchitecturen standaard of via een eenvoudige opt-in naar Blitz te brengen. We hebben hierover een [discussie gaande](https://github.com/blitz-js/blitz/discussions/1841) en we horen het graag als je ideeën hebt.
```

Comment:
```suggestion
Maar we zijn zeer geïnteresseerd in geavanceerdere backendarchitecturen standaard of via een eenvoudige opt-in naar Blitz te brengen. We hebben hierover een [discussie gaande](https://github.com/blitz-js/blitz/discussions/1841) en we horen het graag als je ideeën hebt.
```

Reply:
Waarom is dit in? Om naar Blitz te brengen?

Reply:
Ik denk dat _geïnteresseerd om_ niet zo'n correct Nederlands is, en dat je altijd _geïnteresseerd in_ zegt. Maar ik vind _in_ ook wel niet ideaal in die zin. Dus voor mij mag het in principe ook wel blijven staan hoor.